.github/workflows/publish-wiki.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Wiki
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Upload Documentation to Wiki
+        uses: SwiftDocOrg/github-wiki-publish-action@v1
```

Comment:
This is an outdated action, and will likely stop working after GitHub enforcing some rules (nodejs 16 and set-output removal) recently. Is there any alternative way to publish the wiki?